config.rb

```diff
@@ -101,7 +101,9 @@ def links
 
 catalogue_csv = File.expand_path("data/catalogue.csv", __dir__)
 organisation_csv = File.expand_path("data/organisation.csv", __dir__)
-api_catalogue = ApiCatalogue.from_csv(catalogue_csv: catalogue_csv, organisation_csv: organisation_csv)
+static = ApiCatalogue.from_csv(catalogue_csv: catalogue_csv, organisation_csv: organisation_csv)
+sandbox = ApiCatalogue.from_url("https://federated-api-model-spring-boot-sandbox.london.cloudapps.digital")
```

Comment:
did we want to bring in the ability to load URLs from a CSV instead? Then there's fewer code changes to onboard new API providers?